google-cloud-clients/google-cloud-contrib/google-cloud-logging-logback/src/main/java/com/google/cloud/logging/logback/LoggingAppender.java

```diff
@@ -72,6 +72,7 @@
   private static final String LEVEL_NAME_KEY = "levelName";
   private static final String LEVEL_VALUE_KEY = "levelValue";
   private static final String LOGGER_NAME_KEY = "loggerName";
+  private static final String STACKTRACE = "stacktrace";
```

Comment:
As discussed, instead of `"stacktrace"` can we have a label name with `"fullStacktrace"`?

@ajaaym WDYT?

Reply:
> As discussed, instead of `"stacktrace"` can we have a label name with `"fullStacktrace"`?
> 
> @ajaaym WDYT?

Done